src/__tests__/question.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Question from '../Question';
+
+describe('<Question />', () => {
+  let output;
+  const item = {
+    question: 'Как пройти в библиотеку?',
+    answer: 'Выйти из кабинета и следовать желтым стрелочкам',
+    id: 1,
+  };
+  it('Contain text', () => {
+    output = mount(<Question data={item} />);
```

Comment:
Better divide jsx from js this way:
output = mount(
    <Question data={item} />
);